networking/getting-traffic-into-cluster/getting-traffic-cluster-ingress-controller.adoc

```diff
@@ -0,0 +1,55 @@
+[id="getting-traffic-cluster-ingress-controller"]
+= Getting traffic into a cluster using an Ingress Controller
+include::modules/common-attributes.adoc[]
+:context: getting-traffic-cluster-ingress-controller
+
+toc::[]
+
+
+{product-title} provides methods for communicating from outside the cluster with
+services running in the cluster. This method uses an Ingress Controller.
+
+include::modules/nw-using-ingress-and-routes.adoc[leveloffset=+1]
+
+[NOTE]
+====
+The procedures in this section require prerequisites performed by the cluster
+administrator.
+====
+
+.Prerequisites
+
+Before starting the following procedures, the administrator must:
+
+* Set up the external port to the cluster networking environment so that requests
+can reach the cluster.
+
+* Make sure there is at least one user with cluster admin role. To add this role
+to a user, run the following command:
++
+----
+oc adm policy add-cluster-role-to-user cluster-admin username
+----
+
+* Have an {product-title} cluster with at least one master and at least one node
+and a system outside the cluster that has network access to the cluster. This
+procedure assumes that the external system is on the same subnet as the cluster.
+The additional networking required for external systems on a different subnet is
+out-of-scope for this topic.
+
+include::modules/nw-creating-project-and-service.adoc[leveloffset=+1]
+
+include::modules/nw-exposing-service.adoc[leveloffset=+1]
+
+include::modules/nw-assign-ip-service.adoc[leveloffset=+1]
```

Comment:
This file `nw-assign-ip-service.adoc` is about to assign external IP address to a service, should removed from here.